vcpkg/reference/installation-tree-layout.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Installation tree layout
+description: This article describes the layout conventions used by vcpkg when copying build output into the installation directory.
+author: vicroms
+ms.author: viromer
+ms.date: 9/21/2023
+---
+# Installation tree layout
+
+This article describes the layout conventions used by vcpkg for the packages' installation directory. After each
+package installation, vcpkg copies the produced output into a specific location depending on the
+file's type.
```

Comment:
I think this wording is confusing -- "package installation" can easily be interpreted as "moving from packages/ to installed/" or "What happens when I run `vcpkg install`". This paragraph would then imply that we have technology that reorganizes package contents as some sort of "post-vcpkg install" step by analyzing the file type.